dbt/models/location/access.sql

```diff
@@ -0,0 +1,3 @@
+{{ config(materialized='ephemeral') }}
```

Comment:
I noticed while testing this branch that I had neglected to add a few ephemeral `location` models in https://github.com/ccao-data/data-architecture/pull/64, so the next couple of files fix that omission!